app/javascript/dashboard/routes/dashboard/conversation/contact/ContactInfo.vue

```diff
@@ -290,7 +290,14 @@ export default {
         await this.$store.dispatch('contacts/delete', id);
         this.$emit('panel-close');
         this.showAlert(this.$t('DELETE_CONTACT.API.SUCCESS_MESSAGE'));
-        if (this.$route.name !== 'contacts_dashboard') {
+        if (this.$route.name === 'conversation_through_inbox') {
```

Comment:
@LiamAshdown  Thanks for the PR. Please check the following comments:

1. Let's use the [`isAConversationRoute`](https://github.com/chatwoot/chatwoot/blob/develop/app/javascript/dashboard/routes/dashboard/commands/conversationHotKeys.js#L33) method from conversationHotKeys to check the conversation route. (It would be nice if we could move it to a helper and add specs)
2. Please verify that the logic is correct because deleting a contact still redirects to the contact view.

cc @scmmishra @pranavrajs 

https://github.com/chatwoot/chatwoot/assets/12408980/2baa726f-f33d-4c60-bbac-52c705a4e71b

Reply:
I've updated the pull request. I've created a function and a spec which returns the route dashboard.